Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1580,6 +1580,16 @@ void AXObjectCache::onValidityChange(Element& element)
     postNotification(get(&element), nullptr, AXInvalidStatusChanged);
 }
 
+void AXObjectCache::onTextCompositionChange(Node& node)
+{
+#if ENABLE(ACCESSIBILITY_ISOLATED_TREE)
+    if (auto* object = textCompositionObjectForNode(node))
+        updateIsolatedTree(object, AXTextCompositionChanged);
```

Comment:
Don't need to null-check `object` here (`AXObjectCache::updateIsolatedTree` does it for you). Can do this one-liner instead:

```
updateIsolatedTree(textCompositionObjectForNode(node), AXTextCompositionChanged);
```